mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -71,17 +71,20 @@ static cl::opt<int>
 static cl::opt<std::string> filterLayout("fil_layout",
                                          cl::desc("Filter layout"),
                                          cl::value_desc("layout string"),
-                                         cl::init("kcyx"));
+                                         cl::init("gkcyx"));
```

Comment:
can we have the ability that `g` is NOT specified for layouts for filter / input / weight? And when `g` is NOT specified, treat group count as 1.

Reply:
if we don't specific gkcyx here, we still need to add "g" dim when parsing in other place